usmqe_tests/conftest.py

```diff
@@ -419,6 +418,40 @@ def cluster_reuse(valid_session_credentials):
                     " with FQDN == {}.".format(id_hostname))
 
 
+@pytest.fixture
+def managed_cluster(valid_session_credentials):
+    """
+    Return cluster information from Tendrl API and make sure that returned
+    cluster is *managed*.
+    """
+    cluster_reuse = get_cluster_reuse(valid_session_credentials)
+    if cluster_reuse["is_managed"] != "yes":
+        api = TendrlApi(auth=valid_session_credentials)
+        job_id = api.import_cluster(
+            cluster_reuse["cluster_id"], profiling="enable")["job_id"]
+        api.wait_for_job_status(job_id)
+        return get_cluster_reuse(valid_session_credentials)
+    else:
+        return cluster_reuse
+
+
+@pytest.fixture
+def unmanaged_cluster(valid_session_credentials):
+    """
+    Return cluster information from Tendrl API and make sure that returned
+    cluster is *managed*.
```

Comment:
make sure that returned cluster is *not* managed

Reply:
sorry for pushing, fixed via 24a8d0cf5cb708d4fb0dc416ec69a448a15cd64a